programs/exchange_api/src/exchange_processor.rs

```diff
@@ -15,7 +15,7 @@ pub struct ExchangeProcessor {}
 impl ExchangeProcessor {
     #[allow(clippy::needless_pass_by_value)]
     fn map_to_invalid_arg(err: std::boxed::Box<bincode::ErrorKind>) -> InstructionError {
-        warn!("Deserialze failed: {:?}", err);
+        warn!("Deserialize failed: {:?}", err);
```

Comment:
Better error message here would be nice, why does deserialize fail, is it most likely that the account isn't initialized?